pkgs/applications/graphics/cloudcompare/default.nix

```diff
@@ -31,7 +31,7 @@ mkDerivation rec {
     owner = "CloudCompare";
     repo = "CloudCompare";
     rev = "v${version}";
-    hash = "sha256-KVbFGim2Nqhs5PAQiBNTmJStmnEINTeopiqqEBBniHc=";
+    hash = "sha256-a/0lf3Mt5ZpLFRM8jAoqZer8pY1ROgPRY4dPt34Bk3E=";
```

Comment:
hm, I've just got `sha256-J7Z8cNoH6OkNzz+PhxxRwF03g972w7YDR43KBGrsXUA=`

Reply:
I just tried once again and got `sha256-a/0lf3Mt5ZpLFRM8jAoqZer8pY1ROgPRY4dPt34Bk3E=` .

Reply:
@sikmir , any idea why we hash was different between our computers ?

Reply:
I've run nixpkgs-review and your hash is OK now. So, no idea what it was.